versions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "txIds": {
+    "1723130661028": "XlMDf1o1zFYWwihBxqmgITRtzvvfzjXTBqG132hpwRQ"
+  }
+}
```

Comment:
how is this any more useful than the tag information on the repo?

Reply:
Only in the respect that the ease of access here is simpler but I take your point.